lib/WorseReflection/Core/ServiceLocator.php

```diff
@@ -87,7 +89,14 @@ public function __construct(
         );
 
         $this->sourceLocator = $sourceLocator;
-        $this->docblockFactory = new DocblockParserFactory($this->reflector);
+
+        if (!true) {
```

Comment:
`ServiceLocator` creates parser factory directly in constructor, without access to global configuration (I was thinking of exposing chosen parser to config). I would appreciate any suggestions for making this strategy work. Maybe introduce strategy service, which would be responsible for creating actual factory?

Reply:
hmm, as we need the Reflector to construct it it will either have to be a `boolean` flag to the `ServiceLocator` or a factory so that the factory can be passed in.

The factory is the better choice architecturally, but tbh a boolean flag is also fine.